docs/tutorials/Containers.md

```diff
@@ -65,30 +67,32 @@ requirements:
     DockerRequirement:
         dockerPull: "ubuntu:latest"
 ``` 
-
+___
 ### Docker
 
 [Docker](https://www.docker.com) is a popular container technology that is natively supported by Cromwell and WDL.
 
 
 #### Docker on a Local Backend
 
-On a single machine (laptop or server), no extra configuration is needed to allow docker to run, provided Docker is installed.
+On a single machine (laptop or server), no extra configuration is needed to allow docker to run provided Docker is installed.
 
 You can install Docker for Linux, Mac or Windows from [Docker Hub](https://hub.docker.com/search/?type=edition&offering=community)
 
 #### Docker on Cloud
 
-It is strongly advised that you use Docker images on Cloud backends, because most cloud backends require Docker images to run.
+It is strongly advised that you provider Docker tags to tasks that will run on Cloud backends, and in fact most Cloud providers require it.
```

Comment:
"provider" should be "provide". Also, This section is talking about technologies in a broad sense, I don't think tags need to be discussed here.